homeworks/06-advanced-redux/src/components/TotalPrice.tsx

```diff
@@ -1,14 +1,15 @@
 import React from "react";
-import {PizzaPrice} from "./PizzaPrice";
+import {ShowPizzaPrice} from "./ShowPizzaPrice";
+import {PizzaPrice} from "../types";
 
 interface TotalPriceProps {
-    price: number;
+    price: PizzaPrice;
 }
 
 export function TotalPrice({ price }: TotalPriceProps) {
     return (
         <div className="flex">
-            <span>Total price:</span><PizzaPrice price={price} />
+            <span>Total price:</span><ShowPizzaPrice price={price} />
```

Comment:
I would suggest not using ShowPizzaPrice here otherwise you would need to make lot of changes to he component `ShowPizzaPrice` whenever UI design will change.

Simple `<span>{price}</span>` is good enough here.